content/blog/entries/science-museum-implementation/contents.lr

```diff
@@ -0,0 +1,115 @@
+title: Science Museum provider implementation
+---
+categories: 
+
+cc-catalog 
+gsoc
+gsoc-2020
+---
+author: srinidhi
+---
+series: gsoc-2020-cccatalog
+---
+pub_date: 2020-05-29
+---
+body:
+Science museum is a provider with around 80,000 CC license images, currently Science museum data is ingested from Common Crawl. 
+The problem with Common Crawl is that we don't have control over the content available and this results in poor data quality.
+Science museum is one such provider that has poor data quality and there is need to improve it. So this is done by moving Science museum to an API based crawl. 
+
+## API research
+Science museum provides an official API endpoint using which the implementation will be done, but before the implementation 
+we have to ensure that the API provides necessary content and there is a systematic way to get it. First step is to take an object 
+from their collection and check certain criterias.
+
+[sample object](https://collection.sciencemuseumgroup.org.uk/api/objects/co8005638)
+
+- parameters available for the API 
+- Object landing url
+- Image url
```

Comment:
Please define these.  It's likely the reader won't have the context to understand the difference.